webapp/i18n/en.json

```diff
@@ -1468,7 +1468,7 @@
   "intro_messages.beginning": "Beginning of {name}",
   "intro_messages.channel": "channel",
   "intro_messages.creator": "This is the start of the {name} {type}, created by {creator} on {date}.",
-  "intro_messages.default": "<h4 class='channel-intro__title'>Beginning of {display_name}</h4><p class='channel-intro__content'><strong>Welcome to {display_name}!</strong><br/><br/>This is the first channel teammates see when they sign up - use it for posting updates everyone needs to know.</p>",
+  "intro_messages.default": "<h4 class='channel-intro__title'>Beginning of {display_name}</h4><p class='channel-intro__content'><strong>Welcome to {display_name}!</strong><br/><br/>This is a special channel for communicating to all members in a team. This channel cannot be deleted and members cannot choose to leave this channel.  Because of this, be careful what you add in here, because everyone will see it</p>",
```

Comment:
Thanks for the PR @koddsson! My comment doesn't have to do with your PR specifically, but more so the ticket - I'm wondering if the last sentence comes off a bit too "red-alert." Will queue for discussion.